src/test/scala/scalafix/internal/sbt/JGitCompletionsSuite.scala

```diff
@@ -0,0 +1,27 @@
+package scalafix.internal.sbt
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class JGitCompletionsSuite extends AnyFunSuite {
+
+  test("directory with no .git") {
+    val fs = new Fs()
+    val jgit = new JGitCompletion(fs.workingDirectory)
+    assert(jgit.last20Commits == Nil)
+  }
+
+  test("directory with empty .git") {
+    val fs = new Fs()
+    fs.mkdir(".git")
+    val jgit = new JGitCompletion(fs.workingDirectory)
+    assert(jgit.last20Commits == Nil)
+  }
+
+  test("directory with no commits") {
+    val fs = new Fs()
+    new Git(fs.workingDirectory) // git init
+    val jgit = new JGitCompletion(fs.workingDirectory)
+    assert(jgit.last20Commits == Nil)
+  }
```

Comment:
that's the one reproducing https://github.com/scalacenter/scalafix/issues/1196#issuecomment-654153643 (the other tests pass on master, while this one fails)